airbyte-integrations/connectors/destination-pinecone/metadata.yaml

```
@@ -13,7 +13,7 @@ data:
   connectorSubtype: vectorstore
   connectorType: destination
   definitionId: 3d2b6f84-7f0d-4e3f-a5e5-7c7d4b50eabd
-  dockerImageTag: 0.0.24
+  dockerImageTag: 0.0.25
```

Can we bump to `0.1.0`?

```suggestion
  dockerImageTag: 0.0.25
```